platforms/springboot/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1,4 @@
+io.hawt.springboot.JolokiaEndpointAutoConfiguration
+io.hawt.springboot.HawtioEndpointAutoConfiguration
+io.hawt.springboot.HawtioLogAutoConfiguration
+io.hawt.springboot.HawtioManagementConfiguration
```

Comment:
I'm not knowedgeable about SB 3, but don't we also need the `imports` file for the management feature?

Reply:
See for example:
https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/resources/META-INF/spring/org.springframework.boot.actuate.autoconfigure.web.ManagementContextConfiguration.imports

Reply:
this!! Thanks. :)